src/components/ActivityCreation/ActivityCreationSideBar.tsx

```diff
@@ -50,7 +50,7 @@ const ActivityCreationSideBar = ({
       <H1>新增活動</H1>
       <div className="space-y-6">
         <Link
-          className="group flex items-center gap-2 pl-6"
+          className="group flex w-fit items-center gap-2 pl-6"
           href="#organizer-setting"
```

Comment:
```suggestion
          href="#organizer"
```

Reply:
fix it : 6b854e58eccccd4c67c3c6f573ec6b97a0395010